app/api/items/path_from_root.feature

```diff
@@ -0,0 +1,109 @@
+      | 60 | http://taskplatform.mblockelet.info/task.html?taskId=403449543672183936 | Course  | 1                        | fr                   |
+      | 70 | http://taskplatform.mblockelet.info/task.html?taskId=403449543672183936 | Task    | 1                        | fr                   |
+    And the database has the following table 'items_items':
+      | parent_item_id | child_item_id | child_order |
+      | 10             | 60            | 1           |
+      | 60             | 70            | 1           |
+    And the database has the following table 'items_ancestors':
+      | ancestor_item_id | child_item_id |
+      | 10               | 60            |
+      | 10               | 70            |
+      | 60               | 70            |
+    And the database has the following table 'permissions_generated':
+      | group_id | item_id | can_view_generated       |
+      | 101      | 50      | content                  |
+      | 102      | 10      | content                  |
+      | 102      | 60      | content                  |
+      | 102      | 70      | content                  |
+      | 111      | 10      | content_with_descendants |
+      | 111      | 50      | content_with_descendants |
+    And the database has the following table 'attempts':
+      | id | participant_id | root_item_id | created_at          | ended_at            | allows_submissions_until |
+      | 0  | 101            | null         | 2019-05-30 11:00:00 | null                | 9999-12-31 23:59:59      |
+      | 0  | 102            | null         | 2019-05-30 11:00:00 | null                | 9999-12-31 23:59:59      |
+      | 0  | 111            | null         | 2019-05-30 11:00:00 | null                | 9999-12-31 23:59:59      |
+      | 1  | 102            | 10           | 2019-05-30 11:00:00 | null                | 9999-12-31 23:59:59      |
+      | 2  | 102            | 10           | 2019-05-30 11:00:00 | 2019-05-30 11:00:00 | 9999-12-31 23:59:59      |
+      | 3  | 102            | 10           | 2019-05-30 11:00:00 | null                | 2019-05-30 11:00:00      |
+    And the database has the following table 'results':
+      | attempt_id | participant_id | item_id | started_at          | latest_activity_at  | result_propagation_state |
+      | 1          | 102            | 10      | 2019-05-30 11:00:00 | 2019-05-30 11:00:00 | done                     |
+      | 2          | 102            | 10      | 2019-05-30 11:00:00 | 2019-05-30 11:00:00 | done                     |
+      | 2          | 102            | 60      | 2019-05-30 11:00:00 | 2019-05-30 11:00:00 | done                     |
+      | 3          | 102            | 10      | 2019-05-30 11:00:00 | 2019-05-30 11:00:00 | done                     |
+      | 3          | 102            | 60      | 2019-05-30 11:00:00 | 2019-05-30 11:00:00 | done                     |
+
+  Scenario Outline: Find a path for the user's self group
```

Comment:
I don't get what you mean by "user's self group" here

Reply:
That means 'no as_team_id parameter'

Reply:
I don't get how "user's self group" mean "as a user", but ok ;-)

Reply:
fixed